Idoit.API.Client/CMDB/ObjectTypes/Response/Result.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Newtonsoft.Json;
+
+namespace Idoit.API.Client.CMDB.ObjectTypes.Response
+{
+    public class Result
+    {
+        [JsonProperty("id")]
+        public int id { get; set; }
+        [JsonProperty("title")]
+        public string title { get; set; }
+        [JsonProperty("container")]
+        public string container { get; set; }
+        [JsonProperty("Const")]
+        public string Const { get; set; }
+        [JsonProperty("color")]
+        public string color { get; set; }
+        [JsonProperty("image")]
+        public string image { get; set; }
+        [JsonProperty("icon")]
+        public string icon { get; set; }
+        [JsonProperty("cats")]
+        public string cats { get; set; }
+        [JsonProperty("tree_group")]
+        public string treeGroup { get; set; }
+        [JsonProperty("status")]
+        public string status { get; set; }
+        [JsonProperty("typeGroup")]
+        public string type_group { get; set; }
+        [JsonProperty("type_group_title")]
+        public string typeGroupTitle { get; set; }
```

Comment:
This is a bit confusing. Sometimes the name of an entry includes an underscore, but sometimes it doesn't. I would go a consistent way here so you don't get confused with the variable names later. (tree_group, typeGroup, type_group_title)